src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$selected-link-color: #00acdc;
+$animation-duration: 0.3s;
+$screen-level-1: 488px;
```

Comment:
**Usually** we call these **Breakpoints** of the screen, not _levels_, and 488px is a usual Mobile breakpoint, 768px for Tablets and 1024 for Desktop. 
But this is just an extra knowledge for you to know, don't need to change it ;)